lib/devices/device_descriptors/schwinn_x70.dart

```diff
@@ -61,7 +83,7 @@ class SchwinnX70 extends FixedLayoutDeviceDescriptor with CadenceMixin, PowerSpe
           cadenceMetric: ThreeByteMetricDescriptor(lsb: 4, msb: 6, divider: 1.0),
         ) {
     resistanceMetric = ByteMetricDescriptor(lsb: 16);
-    initCadence(5, 64, maxUint24);
+    initCadence(3, 64, maxUint24);
```

Comment:
Lower 5 -> 3 will lower queue size and faster cadence display response. It could mean less "smooting" of the cadence. Maybe I'll make it configurable in the expert preferences as well.

Reply:
This is also Schwinn x70 specific only. But filed a ticket #538 I'll make it configurable, cannot hurt.